src/celeritas/track/SortTracksAction.cc

```diff
@@ -32,23 +32,25 @@ std::string SortTracksAction::label() const
 /*!
  * Execute the action with host data
  */
-void SortTracksAction::execute(CoreHostRef const& core) const
+void SortTracksAction::execute([[maybe_unused]] ParamsHostCRef const& params,
```

Comment:
Since it's known not to be used...
```suggestion
void SortTracksAction::execute(ParamsHostCRef const&,
```